PDBminer

```diff
@@ -959,11 +926,11 @@ def align_uniprot_pdb(pdb_id, uniprot_sequence, uniprot_numbering, mut_pos, path
     for the b-fcator, mutations in the sequence and the r-free value.
 
     """
-    print("===========================================")
-    print(pdb_id)
-    print("===========================================")
+    logging.debug("===========================================")
+    logging.debug(pdb_id)
+    logging.debug("===========================================")
```

Comment:
I'd rather add the PDB name in the next `logging.debug()` calls - you never know what the order of execution is when using multiple cores so you can't guarantee that you'll have all the lines for a certain PDB in a row